### ISSUE TITLE
Device name not reported when using synced tabs menu with a screen reader

### ISSUE THREAD

**Issue by appsvc-moz** [mozilla/application-services#2276]
STR (with the NVDA screen reader):

1. Ensure you have at least one synced tab on another device.
2. Press the Firefox Account button on the toolbar.
3. Choose Synced Tabs.
4. Press down arrow until you reach the first synced tab.
    - Expected: NVDA should have reported the name of the device the tab was synced from, as well as the name of the tab.
    - Actual: It doesn't; it only reports the tab name.

The best way to fix this would be to make each client a container with role="group" and label it with the device name using aria-labelledby. Unfortunately, right now, there isn't a container element; the buttons are just appended after the label:
https://searchfox.org/mozilla-central/rev/a78233c11a6baf2c308fbed17eb16c6e57b6a2ac/browser/components/customizableui/CustomizableWidgets.jsm#789

I'm not sure whether I can safely add an intervening container here without breaking visual styling (or what element to use), so I can't fix this myself.

---
🐞 Issue is synchronized with Bugzilla [Bug 1598979](https://bugzilla.mozilla.org/show_bug.cgi?id=1598979)

┆Issue is synchronized with this [Jira Story](https://jira.mozilla.com/browse/SYNC-443)
┆Story Points: 3
┆Epic: important not urgent

**Comment by appsvc-moz** [comment]
Upstream bug has been closed with the following resolution: FIXED.
